### DISCUSSION TITLE
How do I escape from lists when "Stick the cursor to the content" is on?

### DISCUSSION THREAD

**Discussion by Calorion** [vslinko/obsidian-outliner#394] [Q&A]
If I'm in a list, and wish to do something other than continue the list, how do I do so when "Stick the cursor to the content" is on? I expect two carriage returns to get me out of the list, but it doesn't.

**Comment by vslinko** [comment]
You can press Enter several times to create a new line without a list mark.


https://user-images.githubusercontent.com/205823/204481612-c779482a-9d9b-407a-801e-da29b4734517.mov

**Comment by Calorion** [comment]
Strange. There was a situation where that wasn't working. Like I said, "I expect two carriage returns to get me out of the list, but it doesn't." I'll have to see if it happens again.
